statistics.md

```diff
@@ -894,9 +896,17 @@ mysql> show warnings;
 
 * [LOAD STATS](/sql-statements/sql-statement-load-stats.md)
 * [DROP STATS](/sql-statements/sql-statement-drop-stats.md)
+* [LOCK STATS](/sql-statements/sql-statement-lock-stats.md)
+* [UNLOCK STATS](/sql-statements/sql-statement-unlock-stats.md)
+* [SHOW STATS_LOCKED](/sql-statements/sql-statement-show-stats-locked.md)
 
 </CustomContent>
 
+<CustomContent platform="tidb-cloud">
+
+* [LOAD STATS](/sql-statements/sql-statement-load-stats.md)
```

Comment:
Is this line duplicated?

Reply:
Do you mean there is one for "tidb" and another for "tidb-cloud"?
![image](https://github.com/pingcap/docs/assets/79440533/2ffd60c8-5aa6-4397-a9ec-d33bedeaa761)

Reply:
Sorry for my mistake. They are different, LOAD vs LOCK